pydantic/fields.py

```diff
@@ -77,6 +77,9 @@ def __init__(self, **kwargs: Any) -> None:
         self.description = kwargs.get('description')
         self.exclude = kwargs.get('exclude')
         self.include = kwargs.get('include')
+        if 'strict' in kwargs and kwargs['strict'] is None:
+            # not set, don't include in schema
+            kwargs.pop('strict')
```

Comment:
This shouldn't be necessary, the schema functions should automatically strip out `None`s.